packages/mobile/src/fiatExchanges/reducer.ts

```diff
@@ -55,13 +23,15 @@ interface TxHashToDisplayInfo {
 }
 
 export interface State {
-  lastUsedProvider: CicoProviderNames | null
+  lastUsedProvider: string | null
```

Comment:
I'm sorry I didn't delete this yet, it's in my TODO list, I'm hoping to do it at some point during this milestone.

Reply:
All good! I was going to do it here but the PR already got kinda big